### ISSUE TITLE
MapSwipe Dev Tutorial Imagery Inconsistency between Web setup and App Tutorial

### ISSUE THREAD

**Issue by nashmsf** [mapswipe/mapswipe#841]
**Describe the bug**
When creating a tutorial in the MapSwipe Dev web interface, the imagery I am using is Bing Imagery. The tasks and references (green, yellow and red where airstrips are/aren't), were based on Bing Imagery used in QGIS and JOSM. So when setting up the task in dev interface online, I can see the tiles match the imagery and instances as they should. However, when actually opening the tutorial in MapSwipe Dev app, the imagery is not synchronous. This has meant that tiles are not correctly identifiable in the tutorial, and confusing what the 'correct' swipes are. See screenshots between two examples across the phone vs setup. 

Perhaps the TMS link for the tutorial on dev app server is not up to date?

**App version**
MapSwipe 2.2.8(1) beta - from this link https://github.com/mapswipe/mapswipe/releases/download/2.2.8(1)-beta/app-dev-release.apk

**Time of crash**
04/04/2024 - 12:55 BST

**Screenshots**
The Android screenshots are the bad(?) images I guess, followed by the setup imagery which is what it should be
![Screenshot_20240404-121436](https://github.com/mapswipe/mapswipe/assets/166012201/89c9834f-9f46-49aa-82e8-b560a3ae3c6a)
![Screenshot 2024-04-04 121442](https://github.com/mapswipe/mapswipe/assets/166012201/95069dec-0e8a-4422-8c12-8bd148f9d623)
![Screenshot_20240404-121331](https://github.com/mapswipe/mapswipe/assets/166012201/8f4d0b46-068f-450b-90ab-e4295b4b14bf)
]
![Screenshot 2024-04-04 121227](https://github.com/mapswipe/mapswipe/assets/166012201/07da1134-bf6d-4b5f-9a7d-267b9fb4a3bf)

**Smartphone (please complete the following information):**
 - Device: Core-X3

 - OS: Android 8.1.0

**Additional context**

**Comment by kopitek8** [comment]
Findings and solution Google Document: https://docs.google.com/document/d/15XCPA93tD1N63psY-bTJ27v1AZCD-t2op63_n-N3imw/edit?usp=sharing

**Comment by nashmsf** [comment]
Side note: In the Google Docs instruction on "Instructions for Ensuring Consistency in the Java OpenStreetMap Editor". The steps indicated there is more suited perhaps to QGIS, not JOSM. 

The imagery URL for Bing uses a different format - which I have not figured out yet how to alter to show different g values (as I think it is accessing the main Bing REST API). 

TLDR: Change "Instructions for Ensuring Consistency in the Java OpenStreetMap Editor" to "Instructions for Ensuring Consistency in the QGIS" in the Google Doc
